components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -28,16 +28,28 @@ object ErrorPages {
             it.readText()
         }
 
+        val showSSLAdvanced: Boolean = when (errorType) {
+            ErrorType.ERROR_SECURITY_SSL, ErrorType.ERROR_SECURITY_BAD_CERT -> true
+            else -> false
+        }
+
         return context.resources.openRawResource(htmlResource)
-            .bufferedReader()
-            .use { it.readText() }
-            .replace("%pageTitle%", context.getString(R.string.mozac_browser_errorpages_page_title))
-            .replace("%backButton%", context.getString(R.string.mozac_browser_errorpages_page_go_back))
-            .replace("%button%", context.getString(errorType.refreshButtonRes))
-            .replace("%messageShort%", context.getString(errorType.titleRes))
-            .replace("%messageLong%", context.getString(errorType.messageRes, uri))
-            .replace("<ul>", "<ul role=\"presentation\">")
-            .replace("%css%", css)
+                .bufferedReader()
```

Comment:
nit: Could you put the indentation back to the original? This breaks our linter.